mmcv/parallel/distributed.py

```diff
@@ -28,6 +28,15 @@ def train_step(self, *inputs, **kwargs):
         ``self.module.forward()`` with ``self.module.train_step()``.
         It is compatible with PyTorch 1.1 - 1.5.
         """
+
+        # In PyTorch >= 1.7, they move the _rebuild_buckets from the end of
```

Comment:
In PyTorch >= 1.7, `reducer._rebuild_buckets()` is moved from the end of backward to the beginning of forward.